### PR TITLE
Quick implementation of container IP records

### PR DESCRIPTION
**Pull request by sttts**
- SRV records according to https://github.com/mesosphere/mesos-dns/pull/197#issuecomment-128594330 and https://docs.google.com/spreadsheets/d/1fZcaUP2Y-MQluMl-JP_PDB1Qlg6Ad36OdSQKBj_zlu4/edit#gid=0
- remove _container prefix and add slave.mesos A records instead for the non-container IP.

### CODE REVIEW

File: records/generator.go [hunk 1]
```diff
@@ -344,42 +343,76 @@ func (rg *RecordGenerator) listenerRecord(listener string, ns string) {
...
+
+			if !task.HasDiscoveryInfo() {
+				continue
+			}
+
+			for _, port := range task.DiscoveryInfo.Ports.DiscoveryPorts {
+				target := canonical + tail + ":" + strconv.Itoa(port.Number)
+
+				// use protocol if defined, fallback to tcp+udp
+				if port.Protocol != "" {
```
Comment (jdef) [inline records/generator.go]: should we attempt to sanitize the `port.Protocol` label before splicing it in?

**Comment by tsenart** [comment]
@jdef: Why did you add the WIP label? Planning to do some work on it?

**Comment by jdef** [comment]
@tsenart yes, I'm going to update docs for A and SRV records

**Comment by tsenart** [comment]
:+1:

File: records/config.go [hunk 2]
```diff
@@ -73,6 +73,34 @@ type Config struct {
...
+		SOAExpire:      86400,
+		SOAMinttl:      60,
+		Resolvers:      []string{"8.8.8.8"},
+		Listener:       "0.0.0.0",
+		HTTPPort:       8123,
+		DNSOn:          true,
+		HTTPOn:         true,
+		ExternalOn:     true,
+		RecurseOn:      true,
+		IPSources:      []string{"docker", "mesos", "host"},
```
Comment (jdef) [inline records/config.go]: the default should not include "docker" since Docker, by default, uses IPs that are not cluster routable

**Pull request merged**